data/migrations/20211102125418_courses.js

```diff
@@ -63,6 +63,6 @@ exports.up = (knex) => {
 exports.down = function (knex) {
   return knex.schema
     .dropTableIfExists('instructors_program_types')
-    .dropTableIfExists('classes')
+    .dropTableIfExists('courses')
```

Comment:
we may not be able to roll back heroku db properly - might need to delete db and run fresh migration